### ISSUE TITLE
Add Long-Press Characters for Serbian/Croatian Letters (č, ć, š, etc)

### ISSUE THREAD

**Issue by m1ndbuzz** [futo-org/android-keyboard#568]
### What happened?

In the release notes of version 0.1.23.2 you mentioned this:
> This fixes a few bugs that were still present in 0.1.23.1:
> - Updated long-press keys for a bunch of languages. The order of important letters should not have changed, and letters not necessary for typing in that language should have moved to the "Misc. letters from common languages" category. This affects Afrikaans, Catalan, Czech, Danish, German, Esperanto, Spanish, Estonian, Finnish, French, Croatian, Hungarian, Icelandic, Italian, Lithuanian, Latvian, Dutch, Polish, Portuguese, Romanian, Slovak, Slovenian, Serbian, Swedish, Turkish. If you type in any of these languages and find errors in the change, i.e. you're missing necessary letters when the misc. letters option is disabled from long-press key layout, please let us know.

but I don't see the letters like *č, ć* when I long press *c* or long pressing *s* I don't get *š* (Serbian/Croatian letters) in the "Misc. letters from common languages" category, even though I have it active.
<img src="https://github.com/user-attachments/assets/df96c863-fd66-4949-a1d4-6d6ae0d15568"  width=50% alt="settings">
<img src="https://github.com/user-attachments/assets/59f51abc-affe-4598-80e9-ad9b85b2e823" width=50% alt="long press s">
<img src="https://github.com/user-attachments/assets/19babdb5-9b6f-4976-8f64-ad0c946c7a0c" width=50% alt="long press c">

Also I have tried all the combinations in the settings to activate/deactivate things and move them around, nothing helped to get the letters to show up on long press.

I am on latest version of GrapheneOS, Pixel 6a. Not sure if that helps.
Let me know if you need any additional information to help you reproduce it.

### Keyboard Version

0.1.23.2

**Comment by z2rec** [comment]
This is a must have feature for me which I loved about gboard. Hope it gets implemented soon, keep up the good work :)

**Comment by m1ndbuzz** [comment]
> This is a must have feature for me which I loved about gboard. Hope it gets implemented soon, keep up the good work :)

I agree. I still use the keyboard daily, and switch to other language when I need those characters (it takes way 10x longer than long press), but can't complain as I really love all the other features that exist.
It replaces Gboard and alternatives almost perfectly.

Great work FUTO team, keep at it. ❤️

**Comment by eldair** [comment]
> This should be fixed by now

Hi, in what version is it fixed? It's still not showing up for me?

**Comment by abb128** [comment]
@eldair You'll have to enable multilingual typing for English and your other language for the long-press characters to be merged. The "Misc. letters from common languages" category is something completely unrelated

**Comment by eldair** [comment]
@abb128 I did that and I still don't see Serbian Latin characters on long press when using English keyboard :/

**Comment by m1ndbuzz** [comment]
It actually works now!
Thank you @abb128

**Comment by Zvonimir-FUTO** [comment]
> [@abb128](https://github.com/abb128) I did that and I still don't see Serbian Latin characters on long press when using English keyboard :/

Could you screenshot your **Languages & Models** settings?

**Comment by eldair** [comment]
@Zvonimir-FUTO sure thing:

<details>
<summary><b>Click to see screenshots</b></summary>

![Screenshot_20260202-161657_FUTO_Keyboard.jpg](https://github.com/user-attachments/assets/c65380f1-8415-482f-b895-6df952a6d5ca)

![Screenshot_20260202-161711_FUTO_Keyboard.jpg](https://github.com/user-attachments/assets/4274684d-cff7-499c-bcd8-be93fcec5b5e)

![Screenshot_20260202-161714_FUTO_Keyboard.jpg](https://github.com/user-attachments/assets/81570395-eed6-4b01-800a-1fafb608067b)

</details>

**Comment by Zvonimir-FUTO** [comment]
@eldair I just tested with the same settings and it seems to work on my side at least.
Could you please open a new bug report for this so we are not spamming everyone in this thread?

https://github.com/futo-org/android-keyboard/issues/new?template=1-bug_report.yml

Please make sure to include screenshot of long key press as well.
